ext/systemd/pxp-agent.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=PCP Execution Protocol (PXP) Agent
+After=syslog.target network.target
+
+[Service]
+EnvironmentFile=-/etc/sysconfig/pxp-agent
+EnvironmentFile=-/etc/default/pxp-agent
+ExecReload=/bin/kill -HUP $MAINPID
```

Comment:
From https://fedoraproject.org/wiki/Packaging:Systemd
"ExecReload= should be specified for all services supporting reload. It is highly recommended to add code here that synchronously reloads the configuration file here (i.e. /bin/kill -HUP $MAINPID is usually a poor choice, due to its asynchronous nature). Omit this option if your service does not support reloading."